indexer/displacement_manager.hpp

```diff
@@ -185,6 +185,7 @@ class DisplacementManager
       // Calculate depth field
       drule::KeysT keys;
       feature::GetDrawRule(ft, zoomLevel, keys);
+      feature::FilterRulesByRuntimeSelector(ft, zoomLevel, keys);
```

Comment:
Consider adding a commentary why we need to rule by a Runtime selector in the index generation, i.e. a phase that precedes runtime.